source/_integrations/spotify.markdown

```diff
@@ -72,6 +72,7 @@ After you have set up the above:
 
 You can send playlists to Spotify using the `"media_content_type": "playlist"`, which are part of the
 [media_player.play_media](/integrations/media_player/#service-media_playerplay_media) service.
+The `media_content_id` can be obtained from the spotify desktop-app by clicking on the more options ("...") next to the album art picture, selecting "share" and then "Copy Spotify URI" or "Copy Playlist Link" (also available in the spotify phone and web app). If you use "Copy Playlist Link" remove everything after and including the question mark '?' in the link.
```

Comment:
```suggestion
The `media_content_id` can be obtained from the Spotify desktop-app by clicking on the more options ("...") next to the album art picture, selecting "share" and then "Copy Spotify URI" or "Copy Playlist Link" (also available in the Spotify phone and web app). If you use "Copy Playlist Link" remove everything after and including the question mark '?' in the link.
```

- Interesting, so leaving in query parameters in the URI doesn't work? If so, that should be fixed in the integration to handle that automatically and nicely. I feel like we should avoid adding notices like that to the docs and make improve the experience instead.

Reply:
@frenck Yes you are correct: leaving in query parameters in the URI doesn't work.
I agree that it would be nicer if the integration itself could just handle it and internally remove everything after the "?" itself. Schould not be to difficult I think.
Could just use a .split("?") or something like that.